tests/users/test_models.py

```diff
@@ -0,0 +1,10 @@
+from tests import factories
+from users.models import User
+
+
+def test_user_model_fullname_fallback():
+    api_data = factories.create_api_user(username="username")
+    api_data["fullname"] = ""
+    User.from_api_data(api_data).fullname == "username"
+    api_data.pop("fullname")
+    User.from_api_data(api_data).fullname == "username"
```

Comment:
Is this some pytest magic, or is this just missing the `assert`s?

Reply:
Ok, so in going bacvk to the fix the missing asserts, I added an extra one, which uncovered a whole other thing, which is fixed in a separate commit.